applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -415,7 +419,7 @@ def ExecuteInitializeSolutionStep(self):
                     if self.interval.IsInInterval(current_time):
                         # We remesh if needed
                         if self.__execute_remesh():
-                            if self.strategy == "Hessian" or self.strategy == "LevelSet":
+                            if self.strategy in ["Hessian", "LevelSet", "Optimization"]:
```

Comment:
```suggestion
                            if self.strategy in ["Hessian", "LevelSet", "optimization"]:
```

Reply:
I was trying to be consistent with the current options, but I guess that it is not a reason to keep doing things out of the style